packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -365,6 +356,7 @@ export class DeltaManager
         private client: IClient,
         private readonly logger: ITelemetryLogger,
         reconnectAllowed: boolean,
+        private readonly container: Pick<Container, "emitDelayInfo" | "cancelDelayInfo">,
```

Comment:
\+ @ChumpChief to share his opinion on layering.
I'd personally prefer all of this logic to be in DeltaManager, not in Container, i.e. better move wrapping of storage into here then other way around. DeltaManager is the right layer to be responsible for all communication with storage

Reply:
Moved all the logic to Delta manager.